docs/pages/2019_MARVEL_Psik_MaX/sections/bands.rst

```diff
@@ -16,10 +16,9 @@ the ``WorkChain``.
 
 The demonstration of the ``WorkChain`` will be performed in a Jupyter
 notebook. To run it, follow the instructions that were given for the
-``QueryBuilder`` notebook in :numref:`querybuilder`. The only difference
-is that instead of selecting the notebook in the ``querybuilder``
-directory, go to ``pw/bandstructure`` and choose the ``bandstructure.ipynb``
-notebook. There you will find some example structures that are loaded from
+``QueryBuilder`` notebook in :numref:`querybuilder`, and 
```

Comment:
I think we can remove `follow the instructions that were given for the QueryBuilder notebook` and just leave "The demonstration of the ``WorkChain`` will be performed in a Jupyter notebook, that you can :download:`download from here <../notebooks/bandstructure.ipynb>`."